tests/conftest.py

```diff
@@ -405,3 +400,12 @@ def pytest_generate_tests(metafunc):
         ):
             params.append("ml.p2.xlarge")
         metafunc.parametrize("instance_type", params, scope="session")
+
+    for fw in ("chainer",):
```

Comment:
Is the tuple with a single string intentional? Do you plan to add other frameworks here later?

Reply:
yep, the idea is to add all the frameworks here and remove their respective individually-defined fixtures